src/init/update-artifacts/docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: '3.6'
 services:
 
   aeproject_node:
-    image: aeternity/aeternity:${NODE_TAG}-bundle
+    image: aeternity/aeternity:${NODE_TAG:-latest}-bundle
```

Comment:
make sense to specify node version as well 🤷‍♀️

Reply:
@davidyuk I agree generally but let's discuss tagging strategy with operations in that regards. I think we should have one major version tag like `6` or `7` which always points to the latest release of that major version. shortly discussed with @thepiwo but didn't ask @dincho or anybody else so far 😅